ui/sortable.js

```diff
@@ -788,12 +789,12 @@ return $.widget("ui.sortable", $.ui.mouse, {
 							.addClass(className || that.currentItem[0].className+" ui-sortable-placeholder")
 							.removeClass("ui-sortable-helper");
 
-					if ( nodeName === "tr" ) {
-						that.currentItem.children().each(function() {
-							$( "<td>&#160;</td>", that.document[0] )
-								.attr( "colspan", $( this ).attr( "colspan" ) || 1 )
-								.appendTo( element );
-						});
+					if ( nodeName === "tbody" ) {
+						tr = $( "<tr></tr>", that.document[0] )
```

Comment:
I'm curious why this is creating a `<tr>` instead of another `<tbody>` to act as a placeholder? As is, sorting `<tbody>`s is kind of weird because the injected `<tr>` used as a placeholder is not the same height as a `<tbody>`.